dist/openrc/zrepl

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+
+command='/usr/local/bin/zrepl'
+command_args='daemon'
+command_background='true'
+pidfile="/run/${RC_SVCNAME}.pid"
+output_log="/var/log/${RC_SVCNAME}.log"
+error_log="/var/log/${RC_SVCNAME}.log"
+
+start() {
+	mkdir -p /var/run/zrepl
+	chmod 750 /var/run/zrepl
```

Comment:
For the systemd file, we use

```
RuntimeDirectory=zrepl zrepl/stdinserver
RuntimeDirectoryMode=0700
```

It creates both `/var/run/zrepl` and `/var/run/zrepl/stdinserver`.
The latter is relevant for the `ssh` transport.
I think the openrc file should create it as well.

Also, systemd auto-removes the runtime directory when stopping the unit.
And it recreates it when restarting the unit.
I don't think zrepl depends on this behavior though - it will re-bind unix socket files if they already exist.

Reply:
@problame Thanks for the feedback, I've fixed both issues (missing stdinserver + runtime directory removal when stopping). Also changed permissions from 750 to 700 to keep it the same as the systemd unit, although I think both are sensible options.